sw/host/opentitanlib/src/app/config/hyperdebug_teacup.json

```diff
@@ -0,0 +1,417 @@
+      "alias_of": "CN10_7"
+    },
+    {
+      "name": "IOC7",
+      "alias_of": "CN10_18"
+    },
+    {
+      "name": "IOC8",
+      "alias_of": "CN8_16"
+    },
+    {
+      "name": "IOC9",
+      "alias_of": "CN10_31"
+    },
+    {
+      "name": "IOC10",
+      "alias_of": "CN10_33"
+    },
+    {
+      "name": "IOC11",
+      "alias_of": "CN10_2"
+    },
+    {
+      "name": "IOC12",
+      "alias_of": "CN10_4"
+    },
+    {
+      "name": "IOR0",
+      "alias_of": "CN7_7"
+    },
+    {
+      "name": "IOR1",
+      "alias_of": "CN7_5"
+    },
+    {
+      "name": "IOR2",
+      "alias_of": "CN7_3"
+    },
+    {
+      "name": "IOR3",
+      "alias_of": "CN7_1"
+    },
+    {
+      "name": "IOR4",
+      "alias_of": "CN7_16"
+    },
+    {
+      "name": "IOR5",
+      "alias_of": "CN9_14"
+    },
+    {
+      "name": "IOR6",
+      "alias_of": "CN9_16"
+    },
+    {
+      "name": "IOR7",
+      "alias_of": "CN9_18"
+    },
+    {
+      "name": "IOR8",
+      "alias_of": "CN9_20"
+    },
+    {
+      "name": "IOR9",
+      "alias_of": "CN9_22"
+    },
+    {
+      "name": "IOR10",
+      "alias_of": "CN9_24"
+    },
+    {
+      "name": "IOR11",
+      "alias_of": "CN9_26"
+    },
+    {
+      "name": "IOR12",
+      "alias_of": "CN9_28"
+    },
+    {
+      "name": "IOR13",
+      "alias_of": "CN9_30"
+    },
+    {
+      "name": "CC1",
+      "alias_of": "CN7_9"
+    },
+    {
+      "name": "CC2",
+      "alias_of": "CN7_10"
+    },
+    {
+      "name": "SPI_DEV_SCK",
+      "mode": "Alternate",
+      "alias_of": "CN10_24"
+    },
+    {
+      "name": "SPI_DEV_D0",
+      "mode": "Alternate",
+      "alias_of": "CN10_23"
+    },
+    {
+      "name": "SPI_DEV_D1",
+      "mode": "Alternate",
+      "alias_of": "CN10_10"
+    },
+    {
+      "name": "SPI1_SCK",
+      "mode": "Input",
+      "alias_of": "CN7_15"
+    },
+    {
+      "name": "SPI1_MOSI",
+      "mode": "Input",
+      "alias_of": "CN7_14"
+    },
+    {
+      "name": "SPI1_MISO",
+      "mode": "Input",
+      "alias_of": "CN7_12"
+    },
+    {
+      "name": "SPI1_CSB",
+      "mode": "Input",
+      "pull_mode": "PullUp",
+      "alias_of": "IOA7"
+    },
+    {
+      "name": "CC1",
+      "mode": "AnalogOutput",
+      "volts": 0.0
+      "alias_of": "CN7_9"
+    },
+    {
+      "name": "CC2",
+      "mode": "AnalogOutput",
+      "volts": 0.0
+      "alias_of": "CN7_10"
+    },
+    {
+      "name": "SW_STRAP0_WEAK",
+      "mode": "PushPull",
+      "level": false,
+      "pull_mode": "None",
+      "alias_of": "CN7_11"
+    },
+    {
+      "name": "SW_STRAP1_WEAK",
+      "mode": "PushPull",
+      "level": false,
+      "pull_mode": "None",
+      "alias_of": "CN10_34"
+    },
+    {
+      "name": "SW_STRAP2_WEAK",
+      "mode": "PushPull",
+      "level": false,
+      "pull_mode": "None",
+      "alias_of": "CN10_20"
+    },
+    {
+      // Controls if CS signal from OT QSPI device is propagated to onboard SPI
```

Comment:
json does not support comments really, should we change the file extension to json5?

Reply:
That sounds as a reasonable refactoring.  I am not familiar with the various supersets of json, or what extension would be the most appropriate.  If there is general agreement that json5 is the extension that matches what our parser accepts, then by all means.
`opentitan_verilator.json` already uses `//` for comments, so I was hoping to merge this as is.

Reply:
Agreed, it should be a future refactoring.